lnwallet/test/test_interface.go

```diff
@@ -1171,7 +1171,7 @@ func testListTransactionDetails(miner *rpctest.Harness,
 			transactions :=
 				make(map[chainhash.Hash][]*wire.TxOut, len(fetchedBlock.Transactions))
 			for _, tx := range fetchedBlock.Transactions {
-				transactions[tx.TxHash()] = tx.TxOut
+				transactions[tx.TxHash()] = tx.Copy().TxOut
```

Comment:
nit: unrelated to the changes in this PR, but if we're touching this, can we format the line above to the following please?

```
			transactions := make(
				map[chainhash.Hash][]*wire.TxOut,
				len(fetchedBlock.Transactions),
			)
```